pages/store/store.css

```diff
@@ -0,0 +1,73 @@
+.main-body {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
+  grid-template-areas: "filter store store store store";
+}
+
+.store-container {
+  grid-area: store;
+  margin: 1rem 0 0 1rem;
+}
+.store-vertical-products {
+  display: flex;
+  gap: 1rem;
+  flex-wrap: wrap;
+}
+.filter-container {
+  grid-area: filter;
+  display: flex;
+  flex-direction: column;
+  gap: 1.2rem;
+  margin-right: 1rem;
+  padding: 2rem;
+  background-color: var(--white);
+}
+.filter-heading {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+.filter-category-container {
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+  font-size: 14px;
+  font-weight: 600;
+}
+.filter-category-checkbox {
+  padding-left: 2rem;
+}
+.filter-rating-container {
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+  font-size: 14px;
+  font-weight: 600;
+}
+.filter-sort-by {
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+  font-size: 14px;
+  font-weight: 600;
+}
+.filter-container input {
+  width: 1rem;
+  height: 1rem;
+  margin-right: 8px;
+}
+.store-heading-container {
+  font-size: 1rem;
+}
+#price-range-slider {
```

Comment:
Any specific reason for using id here ?

Reply:
Actually yes there is a reason when I use class the slider doesn't work but everything work perfectly when I use ID